lib/dialogue-agent/notifications/twilio.ts

```diff
@@ -40,7 +41,8 @@ export default class TwilioNotificationBackend {
     constructor(engine : Engine, config : TwilioConfig) {
         this._platform = engine.platform;
         this._client = new twilio.Twilio(config.accountSid, config.authToken);
-        this._from = config.from;
+        this._from = config.fromByLocale ? (config.fromByLocale[this._platform.locale] || config.from)
+            : config.from;
```

Comment:
just curious, why this feature? someone who has two phone numbers for different languages?

Reply:
Yes, like Covid Genie, which has +1 866 301 0127 for English, and +1 866 301 0129 for Spanish :stuck_out_tongue:

Reply:
oh, this is the agent.. got it